notebooks/models/text-to-text/T5/master_processing.ipynb

```diff
@@ -0,0 +1,530 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "outputs": [],
+   "source": [
+    "import pandas as pd"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-08-03T12:29:30.210120Z",
+     "start_time": "2023-08-03T12:29:24.347729Z"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "outputs": [],
+   "source": [
+    "lecture_list=['ase_combined','corporate_knowledge_management_combined','decision_support_combined', 'information_retrieval_combined','large_scale_combined','leadership_combined','machine_learning_combined','network_science_combined','it-security-all-slides_no_duplicates']"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-08-03T12:29:30.216348Z",
+     "start_time": "2023-08-03T12:29:30.213414Z"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "outputs": [],
+   "source": [
+    "# reading extracted content saved in seperate files.\n",
+    "data_all = []\n",
+    "for file in lecture_list:\n",
+    "    df = pd.read_csv(f'data/{file}.csv')\n",
+    "    df['PDF-Name'] = f'{file}.pdf'\n",
+    "    df['id'] += 1\n",
+    "    data_all.append(df)\n",
+    "content_df = pd.concat(data_all)\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-08-03T12:29:30.430283Z",
+     "start_time": "2023-08-03T12:29:30.219288Z"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "outputs": [],
+   "source": [
+    "master_df = pd.read_csv('master_label.csv',delimiter=';',encoding = 'latin1')"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-08-03T12:29:30.474699Z",
+     "start_time": "2023-08-03T12:29:30.432333Z"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "outputs": [],
+   "source": [
+    "master_df.rename(columns={'Page Number':'id','PDF-Name':'Name'},inplace=True)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-08-03T12:29:30.479208Z",
+     "start_time": "2023-08-03T12:29:30.477792Z"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "outputs": [],
+   "source": [
+    "content_df.rename(columns={'PDF-Name':'Name'},inplace=True)"
```

Comment:
You don't need to rename the column here. You can do it when you read the .csv file

Reply:
done